gatsby_fp/src/components/Layout/Body/Fields/IntellCont/SubHeading.js

```diff
@@ -32,128 +32,97 @@ export default function IntellCont({ intellContList, publicationType }) {
     return sortedIntellContList
   }
 
-    const sortedIntellContList = sortStatusAndYear(intellContList)
+  const sortedIntellContList = sortStatusAndYear(intellContList)
+  const liInnerHtmlList =
+    sortedIntellContList.filter(element => (element.contype === publicationType || element.contypeother === publicationType)).map(element => {
+      let liString = '';
+      if (element.status) {
+        liString += `${element.status} `
+      }
+      if (element.contype && !element.contypeother) {
+        liString += `${element.contype} - `
+      }
+      if (element.contypeother) {
+        liString += `${element.contypeother} - `
+      }
+      element.intellcont_auth.forEach(elem => {
+        if (elem.lname) {
+          liString += `${elem.lname}, `
+        }
+        if (elem.fname && !elem.mname) {
+          liString += `${elem.fname};. `
+        }
+        if (elem.mname && !elem.fname) {
+          liString += `${elem.mname}.; `
+        }
+        if (elem.fname && elem.mname) {
+          liString += `${elem.fname}. ${elem.mname}.; `
+        }
+      })
+      if (element.status === "Published" && element.dty_pub) {
+        liString += `(${element.dty_pub}). `
+      }
+      if (element.status === "Accepted" && element.dty_acc) {
+        liString += `(${element.dty_acc}). `
+      }
+      if (element.status === "Submitted" && element.dty_sub) {
+        liString += `(${element.dty_sub}). `
+      }
+      if (element.title) {
+        liString += `${element.title}. `
+      }
+      if (element.title_secondary) {
+        liString += `${element.title_secondary} `
+      }
+      if (element.issue && !element.pagenum && !element.volume) {
+        liString += `(${element.issue} ed.). `
```

Comment:
The second period here should not be conditional on any of these values. Suppose we have just a title and title_secondary, and we don't have issue/pagenum/volume at all. In this scenario we still need the period to end the sentence.

Reply:
Basically, we want the final period at all times, unless every part of the sentence is missing of course (because then there is no sentence, thus no period necessary).